src/components/Form/FormFactory.js

```diff
@@ -20,7 +20,15 @@ export const getFormElement = props => {
         />
       );
     case 'dropdown':
-      return <Dropdown id={id} value={value || defaultValue} onChange={onChange} onBlur={onBlur} choices={choices} />;
+      return (
+        <Dropdown
+          id={id}
+          value={value || defaultValue}
+          onChange={onChange}
+          onBlur={onBlur}
+          choices={typeof choices === 'function' ? choices() : choices}
```

Comment:
is there a real need for choices to be a function?

Reply:
not anymore :)